README.md

```diff
@@ -145,6 +145,19 @@ click "Show all checks" in the info box at the bottom of the pull request page
 on GitHub, then choose "Details" for the "Test notebooks" job. From the job
 page, click "Summary", then download "Executed notebooks".
 
+## Lint notebooks
```

Comment:
If you take my advise below, you'll want to also mention how to format/fix notebooks. It'd be helpful to mention how CI will have the formatted/fixed notebook for you, too.